contracts/proposals/ccip020-graceful-protocol-shutdown.clar

```diff
@@ -0,0 +1,361 @@
+;; TRAITS
+
+(impl-trait .proposal-trait.proposal-trait)
+(impl-trait .ccip015-trait.ccip015-trait)
+
+;; ERRORS
+
+(define-constant ERR_PANIC (err u2000))
+(define-constant ERR_VOTED_ALREADY (err u2001))
+(define-constant ERR_NOTHING_STACKED (err u2002))
+(define-constant ERR_USER_NOT_FOUND (err u2003))
+(define-constant ERR_PROPOSAL_NOT_ACTIVE (err u2004))
+(define-constant ERR_PROPOSAL_STILL_ACTIVE (err u2005))
+(define-constant ERR_NO_CITY_ID (err u2006)) ;; depracated in favor of MIA_ID and NYC_ID constants
+(define-constant ERR_VOTE_FAILED (err u2007))
+
+;; CONSTANTS
+
+(define-constant SELF (as-contract tx-sender))
+(define-constant MISSED_PAYOUT u1)
+(define-constant CCIP_020 {
+  name: "Graceful Protocol Shutdown",
+  link: "https://github.com/citycoins/governance/blob/feat/add-ccip-020/ccips/ccip-020/ccip-020-graceful-protocol-shutdown.md",
+  hash: "TBD",
+})
+
+(define-constant MIA_ID u1) ;; (contract-call? .ccd004-city-registry get-city-id "mia")
+(define-constant NYC_ID u2) ;; (contract-call? .ccd004-city-registry get-city-id "nyc")
+;; TODO: determine actual percentage from Miami
+(define-constant MIA_DRAW_PERCENTAGE u2) ;; 50% as placeholder
```

Comment:
From the project governance, I think it is still wrong to handle MIA and NYC differently. If 50% belongs to MIA why not the same for NYC?

Reply:
The NYC government never claimed the funds and MIA did, so they are in different situations. I still think they should be separate for reasons listed here:
https://github.com/citycoins/governance/pull/30#issuecomment-2018791406

Reply:
Resolving this convo since the redemption code is removed from the CCIP.